core/new-gui/src/app/workspace/component/property-editor/property-editor.component.ts

```diff
@@ -20,19 +21,46 @@ export type PropertyEditFrameConfig = DynamicComponentConfig<PropertyEditFrameCo
 @UntilDestroy()
 @Component({
   selector: "texera-property-editor",
-  templateUrl: "./property-editor.component.html",
-  styleUrls: ["./property-editor.component.scss"],
+  templateUrl: "property-editor.component.html",
+  styleUrls: ["property-editor.component.scss"],
 })
-export class PropertyEditorComponent implements OnInit {
+export class PropertyEditorComponent implements OnInit, OnDestroy {
   frameComponentConfig?: PropertyEditFrameConfig;
 
+  //加
+  propertyDisplay = true;
+  screenWidth = window.innerWidth;
+  propertyWidth = 400;
+  isOperatorSelected: boolean = false;
+  propertyHeight = 120;
```

Comment:
Can you change the default value propertyHeight to 300 such that the initial height of the panel will not be too small.

Reply:
adjusted